data/unaligned_dataset.py

```diff
@@ -58,16 +58,21 @@ def __getitem__(self, index):
         A_img = Image.open(A_path).convert('RGB')
         B_img = Image.open(B_path).convert('RGB')
 
+        A_path_parts = A_path.split('/')
+        A_seg_path = (A_path_parts[:-2]).join('/') + "trainA_seg/" + A_path_parts[-1]
+        A_seg_img = Image.open(A_seg_path).convert('RGB')
+
         # Apply image transformation
         # For CUT/FastCUT mode, if in finetuning phase (learning rate is decaying),
         # do not perform resize-crop data augmentation of CycleGAN.
         is_finetuning = self.opt.isTrain and self.current_epoch > self.opt.n_epochs
         modified_opt = util.copyconf(self.opt, load_size=self.opt.crop_size if is_finetuning else self.opt.load_size)
         transform = get_transform(modified_opt)
         A = transform(A_img)
+        A_seg = transform(A_seg_img)
```

Comment:
NEed to perform same crop on A and A_seg
https://discuss.pytorch.org/t/torchvision-transfors-how-to-perform-identical-transform-on-both-image-and-target/10606/7